### PR TITLE
OTA-3682: adding migrated meta-updater content

### PR DESCRIPTION
**Pull request by merltron**
Adding content migrated from meta-updater repo.

After this PR is merged, we can merge the equivalent PR in the meta-updater repo: https://github.com/advancedtelematic/meta-updater/pull/595

Apart from "Supported Boards" and "Build Configuration", this content is not referenced in the "developer guide" nav.

It's only linked to from the README of the meta-updater repo (it is just the old content that was always there, but we want to prune out the redundancies and duplicate content in a second pass, before we add anything else to the nav)

Signed-off-by: Merlin Carter <merlin.carter@here.com>

### CODE REVIEW

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-build.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+....
+sudo apt install cpu-checker default-jre parted
+....
+
+To build for https://github.com/advancedtelematic/meta-updater-minnowboard[Minnowboard] with GRUB, you will also need to install https://github.com/tianocore/tianocore.github.io/wiki/OVMF[TianoCore's ovmf] package on your host system. On Debian/Ubuntu, you can do so with this command:
+
+....
+sudo apt install ovmf
+....
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/meta-updater-build.adoc]: It looks like this whole doc can probably be dropped as long as any unique information is preserved elsewhere. I think the first two sections ("Quickstart" and "Dependencies") are probably the most important parts that should be left in the meta-updater README. I'll make that change on the meta-updater PR.

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-dev-config.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| `BRANCH_pn-aktualizr = "mybranch"`
+
+`BRANCH_pn-aktualizr-native = "mybranch"` | Build `mybranch` of Aktualizr. Note that both of these need to be set. This is normally used in conjunction with `require classes/sota_bleeding.inc`
+| `SRCREV_pn-aktualizr = "1004efa3f86cef90c012b34620992b5762b741e3"`
+
+`SRCREV_pn-aktualizr-native = "1004efa3f86cef90c012b34620992b5762b741e3"` | Build the specified revision of Aktualizr. Note that both of these need to be set. This can be used in conjunction with `BRANCH_pn-aktualizr` and `BRANCH_pn-aktualizr-native` but will conflict with `require classes/sota_bleeding.inc`
+| `TOOLCHAIN_HOST_TASK_append = " nativesdk-cmake "` | Use with `bitbake -c populate_sdk core-image-minimal` to build an SDK. See the https://github.com/advancedtelematic/aktualizr#developing-against-an-openembedded-system[aktualizr repo] for more information.
+|======================
+
+== Overriding target version
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/meta-updater-dev-config.adoc]: I agree that most of this doc is redundant, but this "Overriding target version" section is unique IIRC. It's a bit obscure and should just be in some reference section.

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-provisioning-methods.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
+= Manual provisioning
+
+//MC: TOMERGE: Looks mostly like a duplicate of this topic: https://github.com/advancedtelematic/aktualizr/blob/master/docs/ota-client-guide/modules/ROOT/pages/simulate-device-cred-provtest.adoc
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/meta-updater-provisioning-methods.adoc]: Yeah, I think this one is entirely redundant.

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-testing.adoc [hunk 4]
```diff
@@ -0,0 +1,44 @@
+= Testing
+
+//MC: No overlap with any content currently in the developer guide, but probably useful content to clean up and include.
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/meta-updater-testing.adoc]: Agreed.

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-usage.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
+= Usage
+:meta-updater-github-url: https://github.com/advancedtelematic/meta-updater/tree/master
+:metadata-expiry-article: xref:dev@ota-client::metadata-expiry.adoc[OTA Connect documentation]
+ifdef::env-github[:metadata-expiry-article: link:https://docs.ota.here.com/ota-client/dev/metadata-expiry.html[OTA Connect documentation]]
+
+//MC: No overlap with any content currently in the developer guide, but probably useful content to clean up and include. eg: use cases.
```
Comment (patrickvacek) [inline docs/ota-client-guide/modules/ROOT/pages/meta-updater-usage.adoc]: Partially agree. I wonder if anyone has tested the commands in the OSTree section in years. That might not be worth keeping around anymore. And is the garage-push section really not already covered elsewhere? Doesn't seem too useful anymore. "aktualizr configuration" is useful but is largely already covered elsewhere, too. "aktualizr service resource control" might be unique. And the "garage-sign configuration" section was just what I wrote up before it made it to the docs portal; I think that's redundant now, too, right?

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-build.adoc (hunk 1 above)
Comment (merltron) [reply docs/ota-client-guide/modules/ROOT/pages/meta-updater-build.adoc]: Ok, let's say we keep it in there for now and prune it in a second round (just tryna make sure the incoming meta-updater README links work with this PR)

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-dev-config.adoc (hunk 2 above)
Comment (merltron) [reply docs/ota-client-guide/modules/ROOT/pages/meta-updater-dev-config.adoc]: Ditto above

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-provisioning-methods.adoc (hunk 3 above)
Comment (merltron) [reply docs/ota-client-guide/modules/ROOT/pages/meta-updater-provisioning-methods.adoc]: Ditto above

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-testing.adoc (hunk 4 above)
Comment (merltron) [reply docs/ota-client-guide/modules/ROOT/pages/meta-updater-testing.adoc]: Ditto above

File: docs/ota-client-guide/modules/ROOT/pages/meta-updater-usage.adoc (hunk 5 above)
Comment (merltron) [reply docs/ota-client-guide/modules/ROOT/pages/meta-updater-usage.adoc]: Ditto above

**Pull request merged**